src/index.html

```diff
@@ -11,7 +11,192 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>HTML Form</h1>
+
+    <form 
+      action="https://mate-academy-form-lesson.herokuapp.com/create-application" 
+      method="POST"
+    >
+      <fieldset>
+
+        <legend>Personal Information</legend>
+
+        <label  for="surname">Surname:</label>
+        <input type="text" 
+          id="surname" 
+          name="surname" 
+          autocomplete="off" 
+          required
+        ><br>
```

Comment:
check code style
```suggestion
        <input
          type="text" 
          id="surname" 
          name="surname" 
          autocomplete="off" 
          required
        >
```
<img width="914" alt="image" src="https://user-images.githubusercontent.com/102411130/223962645-3caffbb5-e9d7-4ea3-ad4f-a650ccc3ae71.png">

and don't use `br`

Reply:
check all such cases